packages/math/src/math.ts

```diff
@@ -0,0 +1,9 @@
+import { toNumber } from './convert';
```

Comment:
should we flush out other math functions we may need? like `div`, `sub`, `add` and etc?

Reply:
I add multiply and max